<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBSDErrorWindow</class>
 <widget class="QMainWindow" name="WBSDErrorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Errors</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QLabel" name="labelActiveErr">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Active Errors</string>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listActiveErr"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelHistoryErr">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Error History</string>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
         <property name="indent">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadErrorButton">
         <property name="text">
          <string>Load All Error History</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSaveErrors">
         <property name="text">
          <string>Save to file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listHistoryErr"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_Complete_Error_Log">
   <property name="text">
    <string>Save Complete Error Log</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
